docs/schema.md

```diff
@@ -816,36 +817,45 @@ applications.  Here is an entry for a recent Go tool vulnerability:
 
 ```json
 {
-    "schema_version": "1.2.0",
-    "id": "GO-2021-99999",
-    "published": "2021-01-21T19:15:00Z",
-    "modified": "2021-03-10T23:20:53Z",
-    "aliases": ["CVE-2021-3115"],
-    "summary": "packages using cgo can cause arbitrary code execution at build time",
-    "details": "The go command may execute arbitrary code at build time when cgo is in use on Windows. This may occur when running \"go get\", or any other command that builds code. Only users who build untrusted code (and don’t execute it) are affected.\n\nIn addition to Windows users, this can also affect Unix users who have \".\" listed explicitly in their PATH and are running \"go get\" or build commands outside of a module or with module mode disabled.\n\nThanks to RyotaK (https://twitter.com/ryotkak) for reporting this issue.",
-    "references": [
-        {"type": "REPORT", "url": "https://golang.org/issue/43783"}
-    ],
-    "affected": [ {
-        "package": {
-            "ecosystem": "Go",
-            "name": "cmd/go"
-        },
-        "ranges": [
-            {
-                "type": "SEMVER",
-                "events": [
-                    { "introduced": "1.0.0" },
-                    { "fixed": "1.14.14" },
-                    { "introduced": "1.15.10" },
-                    { "fixed": "1.15.17" }
-                ]
-            }
-        ],
-        "ecosystem_specific": {
-            "severity": "HIGH"
+  "id": "GO-2021-0068",
+  "published": "2021-04-14T20:04:52Z",
+  "modified": "2022-05-23T18:04:20Z",
+  "aliases": [ "CVE-2021-3115" ],
+  "details": "The go command may execute arbitrary code at build time when using cgo on Windows.\nThis can be triggered by running go get on a malicious module, or any other time\nthe code is built.\n",
+  "affected": [
+    {
+      "package": {
+        "name": "std",
```

Comment:
The Go security team is still deciding on what this name will be.

Reply:
I've updated everything here to point to Go documentation instead. PTAL.

Reply:
@julieqiu fyi

Reply:
@rsc friendly ping for review, thanks!